src/sentry/conf/server.py

```diff
@@ -134,6 +134,8 @@ def env(key, default="", type=None):
 # here. This directory may not exist until that file is generated.
 CHARTCUTERIE_CONFIG_DIR = os.path.join(DEVSERVICES_CONFIG_DIR, "chartcuterie")
 
+CDC_CONFIG_DIR = os.path.join(DEVSERVICES_CONFIG_DIR, "cdc")
```

Comment:
@lynnagara and I were having some issues getting the snuba config to work (it broke in getsentry's tests).

We narrowed it down to the docker container not having the config file. Maybe @BYK can help point us in the right direction here? When I examined the docker container the only config file that was there was `clickhouse`.

Here is the relevant Dockerfile: https://github.com/getsentry/getsentry/blob/master/Dockerfile.test#L103

Reply:
AFAIK `server.py` is always built into the Sentry package, at least for self-hosted. I'm not familiar with getsentry Docker setup but happy to assist if we can discuss more with more context.

Reply:
Ah sorry, it's not `server.py` that's missing, but rather `./config/` directory where only `./config/clickhouse` gets included.

Reply:
Ah, that might be due to `.dockerignore` or `MANIFEST.in` files where you need to add certain files to an allow-list or explicitly add them.

Reply:
Cool, `MANIFEST.in` sounds right, thanks! @fpacifici let's add the cdc config dir to this file.

cc @lynnagara for snuba

Reply:
Done. Added to MANIFEST